src/library/dlgtrackinfo.cpp

```diff
@@ -42,7 +43,8 @@ DlgTrackInfo::DlgTrackInfo(
         : QDialog(nullptr),
           m_pTrackModel(trackModel),
           m_tapFilter(this, kFilterLength, kMaxInterval),
-          m_pWCoverArtLabel(make_parented<WCoverArtLabel>(this)),
+          m_pWCoverArtMenu(new WCoverArtMenu(this)),
```

Comment:
This can also become make_parented<WCoverArtMenu>

Reply:
Done! :+1: